onnxruntime/core/platform/posix/env.cc

```diff
@@ -469,6 +484,10 @@ class PosixEnv : public Env {
     return val == NULL ? std::string() : std::string(val);
   }
 
+  std::string GetRuntimePath() const override{
+    return where_am_i();
```

Comment:
What's the reason for calling another function as opposed to putting the body of where_am_i here?

Reply:
just make it easier to get the address of function pointer...